configurations/schemas/topics-3.0.1.json

```diff
@@ -0,0 +1,96 @@
+{
+	"description": "A Learning Topic",
+	"bsonType": "object",
+	"required": [
+		"_id",
+		"name"
+	],
+	"properties": {
+		"_id": {
+			"description": "The Topic ID",
+			"bsonType": "objectId"
+		},
+		"status": {
+			"description": "The status of this topic",
+			"msmEnums": "defaultStatus"
+		},
+		"name": {
+			"description": "The Topic Name",
+			"msmType": "msmSentence"
+		},
+		"description": {
+			"description": "The Topic overview paragraph",
+			"msmType": "msmSentence"
+		},
+		"category": {
+			"description": "A simple categorization of topics based on ITIL and ITSM best practices",
+			"msmEnums": "category"
+		},
+		"skills": {
+			"description": "The list of skills related to this topic",
+			"bsonType": "array",
+			"items": {
+				"description": "A Skill",
+				"bsonType": "object",
+				"properties": {
+					"name": {
+						"description": "The short name (slug) for this skill",
+						"msmType": "msmSentence"
+					},
+					"description": {
+						"description": "A description of the skill, a phrase that would go on a resume",
+						"msmType": "msmSentence"
+					},
+					"mastery": {
+						"description": "The level of mastery for this skill, it aligns with person.title",
+						"msmEnums": "mastery"
+					},
+					"speciality": {
+						"description": "The speciality this skill applies to",
+						"msmEnums": "speciality"
+					}
+				},
+				"additionalProperties": false
+			}
+		},
+		"resources": {
+			"description": "List of resources related to this topic",
+			"bsonType": "array",
+			"items": {
+				"description": "A Resource",
+				"bsonType": "object",
+				"properties": {
+					"name": {
+						"description": "The Resource Name",
+						"msmType": "msmSentence"
+					},
+					"link": {
+						"description": "Link to Resource, must be https",
+						"msmType": "msmUrl"
+					},
+					"description": {
+						"description": "The Resource description",
+						"msmType": "msmSentence"
+					},
+					"skills": {
+						"description": "The list of skills (index numbers) this resource contributes to",
+						"bsonType": "array",
+						"items": {
+							"bsonType": "int"
+						}
```

Comment:
Topic skills as defined in this schema no longer have an index number, but resource skills still have a description property referencing it both here and in the corresponding OpenAPI spec in this repo